woocommerce/woocommerce-android
# Creating shipping label: Total card

Part of: #12281

### Description
This PR includes the total card implementation and some margin adjustments. Its changes are focused on UI, and the data used in the screens is mocked. I will connect this UI with real data in a different PR.

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Check that Shipment details card is displayed at the bottom of the screen
6. Tap on/Drag up Shipment details
7. Check that the bottom sheets expand and the Address card is visible

### The tests that have been performed
1. Test that the UI interaction works as expected

### Images/gif

| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://github.com/user-attachments/assets/44a58973-f6ee-4bb8-ad2f-6fa12eee8f38" /> | <img width="300" src="https://github.com/user-attachments/assets/e33c8ed5-a751-429b-acf7-33ee382ad896" />  |


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
